middlewares/authentication.js

```diff
@@ -6,9 +6,7 @@ const { unauthorized } = appRoot.require('errors/errors');
 
 const { username, password } = config.authentication;
 
-/**
- * @summary The middleware for basicAuth
- */
+/** The middleware for basicAuth */
```

Comment:
I guess I'm a bit confused here. When should we use `/** ... */` and when should we use `// ...`?

Reply:
```js
/** comment goes here */
const symbol = {};
```
is a JSDoc comment on `symbol` and is equivalent to
```js
/**
 * comment goes here
 */
const symbol = {};
```

When we want to write a non-JSDoc comment, we should write
```js
// single-line comment goes here
```
for single-line comments and
```js
/*
 * multi-line
 * comment goes here
 */
```
for multi-line comments.

Based on the Airbnb style guide: https://github.com/airbnb/javascript#comments